source/docs/magento1/installation.md

```diff
@@ -38,17 +39,39 @@ composer config http-basic.gitlab.com token $FC_GITLAB_TOKEN
 composer require front-commerce/magento1-module:"dev-master"
 ```
 
+- Enterprise edition version:
+```
+composer config minimum-stability dev
+composer config repositories.front-commerce-ee git https://gitlab.com/front-commerce/magento1-module-enterprise-front-commerce
+composer config repositories.front-commerce git https://gitlab.com/front-commerce/magento1-module-front-commerce.git
+composer config repositories.front-commerce-restful git https://github.com/PH2M/Magento-Extra-RESTful
+composer config http-basic.gitlab.com token $FC_GITLAB_TOKEN
+composer require front-commerce/magento1-module-entreprise:"dev-master"
```

Comment:
I'm wondering if we should keep this `dev-master` spirit here?

Shouldn't we start using tags? If yes, I think that in https://gitlab.com/front-commerce/magento1-module-enterprise-front-commerce/-/blob/5dcd88e47967fe1805fe1fb77f0e6cba0e10d8a4/composer.json#L22 it should be a version too. We can state 1.3.0+ for instance and we'll merge once the 1.3.0 tag is created.